src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.e2e.pages.actions;
+
+import io.cdap.e2e.pages.locators.CdfPluginPropertiesLocators;
+import io.cdap.e2e.pages.locators.CdfSchemaLocators;
+import io.cdap.e2e.pages.locators.CdfStudioLocators;
+import io.cdap.e2e.utils.AssertionHelper;
+import io.cdap.e2e.utils.ConstantsUtil;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.JsonUtils;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.e2e.utils.WaitHelper;
+import org.junit.Assert;
+import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents Cdf Plugin's Properties page Actions
+ */
+public class CdfPluginPropertiesActions {
+  private static final Logger logger = LoggerFactory.getLogger(CdfPluginPropertiesActions.class);
+
+  static {
+    SeleniumHelper.getPropertiesLocators(CdfPluginPropertiesLocators.class);
+    SeleniumHelper.getPropertiesLocators(CdfSchemaLocators.class);
+  }
+
+  public static void clickGetSchemaButton() {
+    ElementHelper.clickOnElement(CdfPluginPropertiesLocators.getSchemaButton);
+    WaitHelper.waitForElementToBeHidden(
+      CdfPluginPropertiesLocators.loadingSpinnerOnGetSchemaButton, ConstantsUtil.SMALL_TIMEOUT_SECONDS);
```

Comment:
Similarly here, get schema might need to interact with various clients to get the schema of the input data and then validate it so this can also take time.

Reply:
Similar fix implemented as for the validate button.